package.json

```diff
@@ -18,6 +18,7 @@
     "clean": "rm -rf ./build && rm -rf ./dist",
     "lint": "eslint .",
     "format": "prettier --write .",
+    "unused-exports": "node scripts/find-unused-exports.js",
```

Comment:
Not related to this PR but I'm a bit concerned to see the number of scripts in `package.json`. It is a lot (24 for now). It might be a good idea to plan a spring cleaning.

Reply:
I'm not opposed to it. Feel free to create a ticket and re-organize/eliminate what you think we can.